sdcm/k8s_configs/eks/scylla-node-prepare.yaml

```diff
@@ -186,6 +186,14 @@ spec:
             value: /mnt/hostfs/mnt/raid-disks/disk0
           - name: SCYLLAD_CONF_MOUNT
             value: /mnt/scylla.d/
+        lifecycle:
+          postStart:
```

Comment:
isn't this racy?
`However, there is no guarantee that the hook will execute before the container ENTRYPOINT` https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/

Reply:
@tnozicka would we have a replacement image anytime soon ? without a fix, we can't run EKS/GKE at all..

Reply:
why don't use use entrypoint==bash and do it there before running the script?

Reply:
It is one more way. I don't mind to redefine the entrypoint.